### ISSUE TITLE
Including intersection_of relationships in obo files

### ISSUE THREAD

**Issue by seger** [obophenotype/xenopus-phenotype-ontology#92]
This would allow Xenbase to associate XPO terms with the PATO/XAO/GO terms from which they were composed and have a way to store it in the Xenbase database. Action for ES: define XPO specific relationships that look like 'has phenotypic entity', 'entity in location' etc. Nico will discuss further with ES.

**Comment by matentzn** [comment]
I am afraid intersection of axioms wont cut it here; because of this 'has part' buffer everywhere, you will need to define relationships instead, like the ones you mention. Lets do this in the next meeting.

**Comment by malcolmfisher103** [comment]
If this issue is going to be very problematic then maybe we should solve it at the Xenbase end rather than requiring you guys to solve it. We really only need these terms for our internal database to facilitate searches, not in the publicly available files. Josh is already parsing the relevant equivalent class terms out of the OWL files to perform his mappings, we can just provide an appropriately formatted file with that information to our Calgary team to marry up with the terms as part of their ontology load. 

We don't want this element to hold up the more important aspects of XPO development. 

If you solved this at your meeting on Wednesday then obviously this comment is redundant.

**Comment by matentzn** [comment]
The last meeting was skipped, we will discuss this at the next meeting. :)

**Comment by matentzn** [comment]
@malcolmfisher103 I did a first stab at this.. 

It is not perfect, but if you could look at it, maybe we can get a sense of where we should go from here.

Check this out:
https://raw.githubusercontent.com/obophenotype/xenopus-phenotype-ontology/master/src/ontology/xpo_xenbase.obo

What I did was:
1) Create a bunch of upheno relations
2) Wrote so-called superclass chains that allow us to materialise these relations from the EQ logical definitions
3) Used a reasoner to materialise the relations..

You will see that there are multiple problems; especially the secondary entity inference is weird. Report all the weird things back to me and I will work on it further with Erik.

**Comment by matentzn** [comment]
Yeah I think I may have some ideas around this.. Lets iterate over this.

Lets start with this one here: What do you like, not like, miss:

```
[Term]
id: XPO:0116438
name: abnormal cardiac chamber development
def: "Abnormal cardiac chamber development." []
is_a: XPO:0116431 ! abnormal anatomical structure development
relationship: UPHENO:9000001 GO:0003205 ! primary affected entity cardiac chamber development
relationship: UPHENO:9000002 GO:0003205 ! secondary affected entity cardiac chamber development
relationship: UPHENO:9000002 GO:0007275 ! secondary affected entity multicellular organism development
relationship: UPHENO:9000002 GO:0007507 ! secondary affected entity heart development
relationship: UPHENO:9000002 GO:0072359 ! secondary affected entity circulatory system development
```
